assessment.md

```diff
@@ -239,17 +239,39 @@ TODO: add description for each assessment level for this skill.
 ### Interactions
 
 * Interaction with the customers
+- SERVICE
+- Social
+- Eager
+- Replace
+- Verbal Communication
+- Informative
+- Countability
+- Easy
 
-TODO: add description for each assessment level for this skill.
+The objective for the interaction with a customer is supposed to be smooth and easy. Interacting with customers is one of the most important parts of software engineering. The acronym of "SERVICE" is a great way for us to see what attributes and tendencies lead to good service.
 
 * Interaction with the team leaders
 
-TODO: add description for each assessment level for this skill.
+- **None**: refuses to interact with and/or work with team leader(s). Ignores advisement or assignment from team leader.
+- **Inadequate**: Interacts with and occasionally works with the team leader(s), but does not actively participate in completing tasks assigned by the team leader(s).
+- **Adequate**: Interacts with and works with the team leader(s) while completing most of the tasks assigned to them by the team leader(s).
+- **Good**: Interacts with, works with, and actively collaborates with the team leader(s). Completes all tasks assigned to them by the team leader(s).
+- **Excellent**: Interacts with, works with, and actively collaborates with the team leader(s). Completes all tasks assigned to them by the team leader(s) and offers additional help to expedite the completion of other related tasks to complete the project.
```

Comment:
The content looks great! One thing to consider is that you should change the scales to include the symbol for each assessment. All groups should agree to use the same symbols to create a standard for this. For example:

Instead of : 
**Adequate**: Interacts with and works with the team leader(s) while completing most of the tasks assigned to them by the team leader(s).

You can change it to this: 
:neutral_face: : Interacts with and works with the team leader(s) while completing most of the tasks assigned to them by the team leader(s).

Other emojis can be decided for each criterion. Additionally, your class could decide to use something other than emojis. It's up to you to decide but you should make sure that other groups agree.